Sources/docc/DocCDocumentation.docc/Reference Syntax/API Reference Syntax/api-reference-syntax.md

```diff
@@ -1,10 +1,10 @@
-# API Documentation
+# API and Conceptual Documentation
```

Comment:
Reading this edit, I wonder if we should have two separate pages, one for API Documentation and one for Conceptual Documentation? Certainly we want to explain and encourage developers to write articles and add them to their API Documentation. So your edits here look good.

What I mean is maybe we need a new page pointing out that developers can write a series of articles which are neither API Documentation nor a tutorial. A series of standalone markdown articles. In this case if you add a new page for that, then you could just keep the original API Documentation title here and avoid the longer title.

Reply:
That was my original idea, but the content would be pretty much the same since the directives that apply to 'API Documentation' are the same as those that can be used for 'Conceptual Documentation,' so it would be like repeating the same curation for both pages.

Reply:
@chuckdude What do you think about 'API and Custom Documentation`? any other title suggestion?

Reply:
> What I mean is maybe we need a new page pointing out that developers can write a series of articles which are neither API Documentation nor a tutorial.

I'm concerned that adding 'Conceptual Documentation' as a new documentation type may be too much of a change. Any thoughts, @patshaughnessy @krilnon @chuckdude?

Reply:
I still concur with @patshaughnessy's original comment that there should be two separate pages or articles to address content. On one hand, you have API reference material that's a doctype of its own. Next, you have the ability to write articles or tutorials. When it comes to articles, those can be task-based, conceptual, or a combination of both, depending on what you need to support people who will use your framework.

So, the three document types are:

* API reference
* Articles
* Tutorials

The page on Articles should address that they can be purely conceptual, strictly task-based, or a combination of the two. The purpose of an article is to teach developers how to use the API, or to show how to do something (task-based) in a practical sense.

I don't think we should try to wedge API reference and Articles (or Tutorials) into the same article.